README.md

```diff
@@ -89,6 +89,7 @@ Use these hashtags in search to filter out the tools
 
 - [Easy-Peasy.ai](https://easy-peasy.ai) - AI copywriting tools to help you tell your story in the most engaging way. #freemium
 - [PixAI.Art](https://pixai.art) - AI art generator designed specifically for creating anime-inspired artwork #free
+- [Dall-E 3](https://openai.com/dall-e-3) - AI image generator accesible Microsoft's Bing Chat and Bing Image Creator #free
```

Comment:
Can you sort this alphabetically? place it above Easy-Peasy.ai. Thank you!

Reply:
Sure!

Reply:
Done!